src/devtools/client/debugger/src/reducers/sources.js

```diff
@@ -664,4 +665,9 @@ export function getTextAtLocation(state, id, location) {
   return text;
 }
 
+export function getIsSourceMappedSource(state) {
+  const sourceId = getSelectedSourceId(state);
+  return ThreadFront.isSourceMappedSource(sourceId);
```

Comment:
It feels a little dirty to be calling into ThreadFront from a reducer like this. It would be cleaner to have the source have an `isOriginal` field that we could check. e.g. `return source.isOriginal`

Reply:
👍